### PR TITLE
Fix :recognize-object state to return t or nil

### PR DESCRIPTION
**Pull request by pazeshun**
From https://github.com/start-jsk/jsk_apc/pull/2661#pullrequestreview-157527987
Merge this PR or #2661 
Tested in real robot

### CODE REVIEW

**Comment by knorth55** [comment]
I prefer this PR

File: jsk_arc2017_baxter/euslisp/pick-main.l [hunk 1]
```diff
@@ -74,7 +74,7 @@
             (:recognize-object
               '(lambda (userdata)
                  (send *ti* :update-state *arm* :recognize-object)
-                 (send *ti* :recognize-object *arm* :trial-times 3)))
+                 (not (null (send *ti* :recognize-object *arm* :trial-times 3)))))
```
Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/pick-main.l]: `(if (send *ti* :recognize-object *arm* :trial-times 3) t nil)` is easy to understand

Comment (pazeshun) [reply jsk_arc2017_baxter/euslisp/pick-main.l]: Thanks. Updated.

**Comment by pazeshun** [comment]
Rebase origin/master

**Comment by pazeshun** [comment]
Test passed.
@knorth55 Could you review and merge this?

**Pull request merged**